narayana/ArjunaJTA/jta/tests/classes/org/jboss/narayana/rts/EmptyTxnTest.java

```diff
@@ -50,15 +49,19 @@ public static void tearDown() throws IOException {
 
     @Benchmark
     public void testEmptyTxn(Blackhole bh) throws IOException {
-        bh.consume(beginTx());
+        Set<Link> links = beginTx();
```

Comment:
You are right, I will restore the consume call. It is my mistake related to a wrong understanding of the consume call.

Reply:
In order to address your request I made use of a List in order to store the links. In fact inside the consume method we can only call for method returning boolean. Do you think it is ok?

Reply:
Sounds good.

Reply:
Did you mean Set instead of List. The change here to a Set is fine if that is what you mean?

Reply:
Sorry @mmusgrov , at a certain point I implemented the List. Though I did not like to use that extra structure in the benchmark test. So I changed my mind and I then decided to avoid using the List, and I moved back using the Set structure as it was before. Eventually the Blackhole.consume() call take as a parameter the return value of a support method (where I manage to implement the assignations I need). I hope it is clear :+1: